ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -72,25 +25,29 @@ import UIKit
     var size: MSFActivityIndicatorSize { get set }
 }
 
-/// View that represents the activity indicator
-public struct ActivityIndicator: View {
-    @Environment(\.theme) var theme: FluentUIStyle
-    @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
-    @ObservedObject var tokens: MSFActivityIndicatorTokens
-    @ObservedObject var state: MSFActivityIndicatorStateImpl
-    @State var rotationAngle: Double = 0.0
+/// View that represents the Activity Indicator.
+public struct ActivityIndicator: View, TokenizedControlInternal {
+    public let tokenKey: String
 
     /// Creates the ActivityIndicator.
     /// - Parameter size: The MSFActivityIndicatorSize value used by the Activity Indicator.
     public init(size: MSFActivityIndicatorSize) {
         let state = MSFActivityIndicatorStateImpl(size: size)
         self.state = state
-        self.tokens = state.tokens
+
+        // Creates separate lookup keys for all different sizes.
+        self.tokenKey = "\(type(of: self))_\(size.rawValue)"
     }
 
     public var body: some View {
-        let size = tokens.activityIndicatorSize
-        let color = Color(state.color ?? tokens.defaultColor)
+        let side = tokens.side
```

Comment:
nit: side seems like an odd name to me. I get that we're trying to not have too many things named size, but I think I'd prefer length or something.

Reply:
The reasoning behind the name is that for rectangular frames we have different values for height and width. The term in geometry for the height and width of a square is "side" which is why I chose to use it. It implies that the shape of the frame is a square of equal sides.